### PR TITLE
Implement invite codes

### PR DESCRIPTION
**Pull request by josefarias**
/claim #259

Closes https://github.com/maybe-finance/maybe/issues/259

### CODE REVIEW

**Comment by Shpigford** [comment]
@josefarias Looking good! Is `rails c` then `InviteCode.generate!` the way to generate codes?

**Comment by josefarias** [comment]
> @josefarias Looking good! Is `rails c` then `InviteCode.generate!` the way to generate codes?

@Shpigford correct!

File: app/views/layouts/auth.html.erb [hunk 1]
```diff
@@ -18,15 +18,21 @@
   </head>
 
   <body class="h-full">
+    <% flash.each do |type, msg| %>
+      <div>
+        <%= msg %>
+      </div>
+    <% end %>
```
Comment (josefarias) [inline app/views/layouts/auth.html.erb:25]: Should probably style these. Could do that in this PR or separately.

**Comment by Shpigford** [comment]
I wouldn't worry about the message styling. We need to overhaul that anyways.

Can you add a rake task for the invite code (so we don't even have to hop in to console)?

**Comment by josefarias** [comment]
> I wouldn't worry about the message styling. We need to overhaul that anyways.

Sounds good!

> Can you add a rake task for the invite code (so we don't even have to hop in to console)?

Done! `bin/rake invites:create`

**Pull request merged**